src/mapkit.cs

```diff
@@ -1953,7 +1951,7 @@ interface MKLocalSearchCompleterDelegate {
 
 	[TV (9,2)][NoWatch][iOS (9,3)]
 	[BaseType (typeof(NSObject))]
-#if MONOMAC || XAMCORE_3_0 // Avoid breaking change in iOS
+#if MONOMAC || XAMCORE_3_0 // "You do not create instances of this class directly"
```

Comment:
<Boromir>"One does not simply create instances of this class directly"</Boromir>

Reply:
--Boromir